README.md

```
@@ -2,41 +2,89 @@
+
+### Model providers and examples
+
+- **OpenAI**: `gpt-4o`, `gpt-4o-mini`, `gpt-5`, `gpt-5-mini`, `o3`, `o4-mini`
+- **Anthropic**: `claude-opus-4-1`, `claude-sonnet-4`, `claude-3-5-sonnet`
+- **OpenRouter**: curated/free models via OpenRouter
+
+---
+
+## Monorepo layout
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)